doc/user-guides/full-walkthrough/secure-protect-connect-openshift.md

```diff
@@ -1,696 +1,515 @@
+Check the status of the health checks as follow:
 
 ```bash
-export oasPath=examples/oas-apikey.yaml
-# Ensure you still have these environment variables setup from the start of this guide:
-export rootDomain=example.com
-export gatewayNS=api-gateway
+kubectl get dnspolicy ${KUADRANT_GATEWAY_NAME}-dnspolicy -n ${KUADRANT_GATEWAY_NS} -o=jsonpath='{.status.conditions[?(@.type=="SubResourcesHealthy")].message}'
 ```
-
-### Step 3 - Use OAS to define your HTTPRoute rules
-
-You can generate Kuadrant and Gateway API resources directly from OAS documents by using an `x-kuadrant` extension.
-
-NOTE: For a more in-depth look at the OAS extension, see the [kuadrantctl documentation](https://docs.kuadrant.io/latest/kuadrantctl/).
-
-You will use `kuadrantctl` to generate your `HTTPRoute`.
-
-NOTE: The sample OAS has some placeholders for namespaces and domains. You will inject valid values into these placeholders based on your previous environment variables.
-
-Generate the resource from your OAS as follows, (`envsubst` will replace the placeholders):
+### Test the `low-limit` and `deny all` policies
 
 ```bash
-cat $oasPath | envsubst | kuadrantctl generate gatewayapi httproute --oas - | kubectl apply -f -
+while :; do curl -k --write-out '%{http_code}\n' --silent --output /dev/null  "https://api.$KUADRANT_ZONE_ROOT_DOMAIN/cars" | grep -E --color "\b(429)\b|$"; sleep 1; done
 ```
 
-```bash
-kubectl get httproute toystore -n ${devNS} -o=yaml
-```
+### (Optional) Configure metrics to be scraped from the Gateway instance
 
-You should see that this route is affected by the `AuthPolicy` and `RateLimitPolicy` defined as defaults on the Gateway in the Gateway namespace.
-
-```yaml
-- lastTransitionTime: "2024-04-26T13:37:43Z"
-        message: Object affected by AuthPolicy demo/external
-        observedGeneration: 2
-        reason: Accepted
-        status: "True"
-        type: kuadrant.io/AuthPolicyAffected
-- lastTransitionTime: "2024-04-26T14:07:28Z"
-        message: Object affected by RateLimitPolicy demo/external
-        observedGeneration: 1
-        reason: Accepted
-        status: "True"
-        type: kuadrant.io/RateLimitPolicyAffected        
-```
-
-### Step 4 - Test connectivity and deny-all auth 
-
-You can use `curl` to hit an endpoint in the toystore app. Because you are using Let's Encrypt staging in this example, you can pass the `-k` flag as follows:
+If Prometheus is installed on the cluster, set up a `PodMonitor` to configure it to scrape metrics directly from the Gateway pod.
+This must be done in the namespace where the Gateway is running. For a list of the metrics you'll get see the Kuadrant [docs](https://docs.kuadrant.io/0.11.0/kuadrant-operator/doc/observability/metrics/)
 
 ```bash
-curl -s -k -o /dev/null -w "%{http_code}" "https://$(kubectl get httproute toystore -n ${devNS} -o=jsonpath='{.spec.hostnames[0]}')/v1/toys"
+kubectl apply -f - <<EOF
+apiVersion: monitoring.coreos.com/v1
```

Comment:
I don't feel like this actually adds that much and is a bit jarring in the flow. Might make more sense in an observability walkthrough?

Reply:
sure ill remove it